Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
```

Comment:
like I said before, the system-proxies will only make qt use system proxies settings, without it users will need to use qt api to set proxy settings. I doubt any developer want their users to set proxy ONLY for qt.

and the pch, this settings is default on windows build, but never get to be default on any other platform, because, the msvc always support pch, but not all other compiler support it, I remember that gcc supports pch start with version 3.4. so the pch is never default setting except under windows with msvc.